static/grunticon.embed.cors.js

```diff
@@ -25,16 +25,33 @@
 		}
 		grunticon.ready(function(){
 			ajaxGet( grunticon.href, function() {
-				var style = document.createElement( "style" );
-				style.innerHTML = this.responseText;
+				// check to see if we've already created a style block for this href
+				var existingStyle = window
+					.document
+					.querySelector( 'style[data-href$="'+ grunticon.href +'"]' );
 				var ref = grunticon.getCSS( grunticon.href );
-				if( ref ){
+
+				if( existingStyle ){
+					// NOTE this will do nothing at all if the embeds are already there
+					// and the embed attribute has been removed. The only purpose for this
+					// is if the markup has been "rerendered" in the page after the
+					// initial embed at which point the embed attributes should exist and
+					// the existing embeds should be gone
+					grunticon.embedIcons( grunticon.getIcons( existingStyle ) );
+				} else {
```

Comment:
This should probably be `else if ( ref ) {` since it's not for sure that there will be a link matching the `href`